src/services/refactors/convertFunctionToEs6Class.ts

```diff
@@ -183,20 +186,42 @@ namespace ts.refactor {
                             const expression = arrowFunctionBody as Expression;
                             bodyBlock = createBlock([createReturn(expression)]);
                         }
-                        return createMethod(/*decorators*/ undefined, modifiers, /*asteriskToken*/ undefined, memberDeclaration.name, /*questionToken*/ undefined,
+                        const method = createMethod(/*decorators*/ undefined, modifiers, /*asteriskToken*/ undefined, memberDeclaration.name, /*questionToken*/ undefined,

```

Comment:
Not related to comments, but do we preserve the binding to the `this` of the outer scope if the arrow contains a `this`? What happens if we refactor:

```js
function C() {}
C.prototype.x = () => this;
```